src/pages/Environments.js

```diff
@@ -594,13 +594,13 @@ export const Environments = ({ nav = undefined }) => {
           span({ style: { fontWeight: 600 } }, [' Hide resources you did not create'])
         ])
       ]),
-      runtimes && h(SimpleFlexTable, {
+      runtimes && div({ style: { overflow: 'scroll' } }, [h(SimpleFlexTable, {
```

Comment:
Nit: bring h(SimpleFlex...) to the next line, and add a new line to separate the `])` for the div below (Current line 827)